src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -160,6 +163,53 @@ export const PaymentContent: FC<Props> = props => {
         />
       </Collapse>
 
+      {/* Sepa debit */}
+      <Collapse open={paymentMethod === "SEPA_DEBIT"}>
+        <Flex>
+          <Text color="black60" variant="sm">
+            • Your bank account must be located in one of the SEPA
+            member-states.
+          </Text>
+          <Tooltip
+            placement="top-start"
+            size="lg"
+            width={400}
+            content={
+              <>
+                <Text variant="sm">SEPA member-states</Text>
+                <Spacer mb={2} />
+                <Text variant="sm">
+                  Austria, Belgium, Britain, Bulgaria, Croatia, Cyprus, Czech
+                  Republic, Denmark, Estonia, Finland, France, Germany, Greece,
+                  Hungary, Iceland, Ireland, Italy, Latvia, Liechtenstein,
+                  Lithuania, Luxembourg, Malta, Monaco, Netherlands, Norway,
+                  Poland, Portugal, Romania, Slovakia, Slovenia, Spain, Sweden,
+                  and Switzerland.
+                </Text>
+              </>
+            }
+          >
+            <Clickable ml={0.5} style={{ lineHeight: 0 }}>
+              <InfoCircleIcon />
+            </Clickable>
+          </Tooltip>
```

Comment:
**chore-ish**: I think this is not too critical, but the tooltip in the designs looks wider (although it's 400px wide as well, it has 7 countries in the first line for example). wonder if it's the font size 🤔